### DISCUSSION TITLE
Navbar Sticky Top Not Working

### DISCUSSION THREAD

**Discussion by digidoda** [holger1411/understrap#2167] [Q&A]
Hi all - I'm trying to make my navbar sticky-top but it doesn't seem to work properly so I was wondering if there was something I was doing wrong?  If I give the navbar a 'fixed-top' class that works but then the content is pulled underneath the navbar and I would prefer to have it use sticky if possible if anyone had any pointers on getting that working properly please?

**Comment by 1zboro1** [comment]
Having the same problem. I tried adding position: sticky with top: 0 but nothing happens. Even removing all the overflow: hidden on the site doesn't change a thing.

**Comment by IanDelMar** [comment]
For `.sticky-top` to work, you need to move the navbar outside the header.

**Reply by digidoda** [reply]
I think I've found the answer - if you add sticky top to the navbar area in the header.php file instead of editing the offcanvas / collapse file then the sticky top seems to work.

In my header.php file it was on line 32 so I've adapted the code to be the following:

<header id="wrapper-navbar" class="sticky-top shadow border-bottom border-white">

That seems to make it all work as expected

**Reply by digidoda** [reply]
`<header id="wrapper-navbar" class="sticky-top shadow border-bottom border-white">`

**Reply by IanDelMar** [reply]
Yes, this will work if you do not have any content in the header that you do not want to be sticky.
